react-project-aston/src/pages/Favourites.js

```diff
@@ -1,5 +1,22 @@
+import { useSelector } from 'react-redux';
+
+import BookList from '../components/BookList';
+import Wrapper from '../components/Layout/Wrapper';
+
 const Favourites = () => {
-  return <h1>I am a favourites page</h1>;
+  const favourites = useSelector((state) => state.favourites.favourites);
+  const isNotEmpty = favourites.length > 0;
```

Comment:
Флаги лучше называть со стороны позитивного кейса в большинстве случаев. Чтобы потом не делать Отрицание на отрицательное, как у тебя ниже !isNotEmpty https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig
```suggestion
  const isEmpty = favourites.length === 0;
```